packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -167,13 +166,13 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
   /// callback that returns a string describing useful debugging information,
   /// e.g. the arguments passed to the callback.
   @protected
-  T invokeCallback<T>(String name, RecognizerCallback<T> callback, { String debugReport() }) {
+  T? invokeCallback<T>(String name, RecognizerCallback<T> callback, { String Function()? debugReport }) {
```

Comment:
I am surprised that the return type can be null here.

Reply:
It can be null because the catch block doesn't throw and then the result remains null